tests/framework/results/test_manager.py

```diff
@@ -0,0 +1,102 @@
+from types import MethodType
+
+import pytest
+
+from vivarium.framework.results.manager import ResultsManager
+
+from .mocks import CATEGORIES, NAME, SOURCES, sorting_hat_serial, sorting_hat_vector
+
+
+# Mock for get_value call for Pipelines, returns a str instead of a Pipeline
+def mock_get_value(self, name: str):
+    return name
+
+
+def verify_stratification_added(mgr, name, categories, sources, mapper, is_vectorized):
```

Comment:
I misread this and thought it was a test. Maybe add a comment to clarify it is not. 

Also, consider whether we want to define the `__eq__` function on `Stratification` to simplify this check. 

In `test_context.py` line 41 you make the assertion `assert expected_object in ctx._stratifications`, and I thought that `in` keyword is implicitly doing an equality check, so check if the default equality function is sufficient for this purpose and that the above test is correct.

Reply:
I recall why I did it the two different ways, but in retrospect, I don't think my reasoning was right. I'm going to refactor the more deliberate `verify_stratification_added` to be used by both `test_manager.py` and `test_context.py`